src/test/java/core/basesyntax/service/impl/ParserImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.service.Parser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserImplTest {
+    private static Parser parser;
+
+    @BeforeClass
+    public static void beforeClass() {
+        parser = new ParserImpl();
+    }
+
+    @Test
+    public void test_Parser_Ok() {
+        List<String> strings = new ArrayList<>();
+        strings.add("type,fruit,quantity");
+        strings.add("s,banana,150");
+        FruitDto expected = new FruitDto("s", "banana", 150);
+        FruitDto actual = parser.parseLines(strings).get(0);
+        assertEquals(expected, actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void test_ParserCount_NotOk() {
```

Comment:
```suggestion
    public void parseLines_NegativeQuantity_NotOk() {
```
let's make all methods in all test classes follow the same naming pattern
method_condition_result